src/BlockDecomposition.jl

```diff
@@ -43,6 +43,14 @@ function model_factory(::Val{false}, args...; kw...)::JuMP.Model
     return JuMP.Model(args...; kw...)
 end
 
+"""
+    BlockModel(optimizer [, direct_model = false])
+
+Declare a JuMP hooked by the method that register the decomposition done by the user.
```

Comment:
Is "model" missing here? I couldn't understand well "a JuMP hooked". Also I'm not sure if it's possible to have a custom method that registers the decomposition. But if it's not, maybe "the BlockDecomposition method" would be more clear to me (just to be sure this method isn't an argument). I don't know if that makes sense in a BlockDecomposition docstring.

Reply:
```suggestion
Return a JuMP model which BlockDecomposition will decompose using instructions given by the user.
```